internal/dao/helm.go

```diff
@@ -58,6 +59,24 @@ func (c *Helm) Get(_ context.Context, path string) (runtime.Object, error) {
 	return render.HelmRes{Release: resp}, nil
 }
 
+// GetValues returns values of a release
+func (c *Helm) GetValues(path string, allValues bool) (string, error) {
+	ns, n := client.Namespaced(path)
+	cfg, err := c.EnsureHelmConfig(ns)
+	if err != nil {
+		return "", err
+	}
+	vals := action.NewGetValues(cfg)
+	vals.AllValues = allValues
+	resp, err := vals.Run(n)
+	if err != nil {
+		return "", err
+	}
+	yamlBytes, err := yaml.Marshal(resp)
+	yamlString := string(yamlBytes)
```

Comment:
Might be better off just returning the slice of bytes here vs a string and let downstream handle it.